tests/examples/examples_test.go

```diff
@@ -164,7 +164,7 @@ func TestAccHelmLocal(t *testing.T) {
 				t *testing.T, stackInfo integration.RuntimeValidationStackInfo,
 			) {
 				assert.NotNil(t, stackInfo.Deployment)
-				assert.Equal(t, 15, len(stackInfo.Deployment.Resources))
+				assert.Equal(t, 16, len(stackInfo.Deployment.Resources))
```

Comment:
Curious - why are all of these changing? Seems like there is a new resource being created after these changes - but unclear why that should be?

Reply:
It's creating an extra default provider reference due to the invoke.